docker/docker-compose.yml

```diff
@@ -119,7 +119,8 @@ services:
   # explicitly wait for this container
   kafka-setup:
     build:
-      context: kafka-setup
+      dockerfile: ./docker/kafka-setup/Dockerfile
```

Comment:
This should also be fixed on the neo4j yml

Reply:
Thanks